spec/system/comments/user_edits_a_comment_spec.rb

```diff
@@ -10,6 +10,7 @@
            user: user,
            body_markdown: Faker::Lorem.paragraph)
   end
+  let(:iso8601_datetime_regexp) { /^((\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2})Z)$/ }
```

Comment:
Wondering if we should make this a constant for testing since we are using it in a couple of places.

Reply:
Thought about it, got lost in RSpec's documentation, was almost at the 15 minute mark, thought that we need a third instance to refactor :D